src/Framework/TestCase.php

```diff
@@ -166,7 +166,7 @@
     /**
      * The name of the expected Exception.
      *
-     * @var string
+     * @var null|string
```

Comment:
it's not initialized with any value, if one would not call `expectException`, then this would remain null.